shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/screens/MainScreen.kt

```diff
@@ -81,8 +88,13 @@ fun MainScreen(
             modifier = Modifier
                 .fillMaxWidth()
                 .weight(1f),
+            enterTransition = { EnterTransition.None },
+            exitTransition = { ExitTransition.None },
+            popEnterTransition = { fadeIn() },
+            popExitTransition = { fadeOut() },
         ) {
             composable<InfoScreen> {
+                MainBackHandler()
```

Comment:
It's weird to do this manually in every screen

Reply:
Unfortunately the handler has to be inside the nav destination for it to capture back gestures from there. At least we can do it here in the nav graph setup instead of having to put it directly inside each screen.